pkg/resources/grant_privileges_to_account_role.go

```diff
@@ -0,0 +1,1109 @@
+		Required:         true,
+		ForceNew:         true,
+		Description:      "The fully qualified name of the account role to which privileges will be granted.",
+		ValidateDiagFunc: IsValidIdentifier[sdk.AccountObjectIdentifier](),
+	},
+	"privileges": {
+		Type:        schema.TypeSet,
+		Optional:    true,
+		Description: "The privileges to grant on the account role.",
+		ExactlyOneOf: []string{
+			"privileges",
+			"all_privileges",
+		},
+		Elem: &schema.Schema{
+			Type:             schema.TypeString,
+			ValidateDiagFunc: isNotOwnershipGrant(),
+		},
+	},
+	"all_privileges": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "Grant all privileges on the account role.",
+		ExactlyOneOf: []string{
+			"privileges",
+			"all_privileges",
+		},
+	},
+	"with_grant_option": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		ForceNew:    true,
+		Description: "Specifies whether the grantee can grant the privileges to other users.",
+	},
+	"always_apply": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		Description: "If true, the resource will always produce a “plan” and on “apply” it will re-grant defined privileges. It is supposed to be used only in “grant privileges on all X’s in database / schema Y” or “grant all privileges to X” scenarios to make sure that every new object in a given database / schema is granted by the account role and every new privilege is granted to the database role. Important note: this flag is not compliant with the Terraform assumptions of the config being eventually convergent (producing an empty plan).",
+	},
+	"always_apply_trigger": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "",
+		Description: "This is a helper field and should not be set. Its main purpose is to help to achieve the functionality described by the always_apply field.",
+	},
+	"on_account": {
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Default:     false,
+		ForceNew:    true,
+		Description: "If true, the privileges will be granted on the account.",
+		ExactlyOneOf: []string{
+			"on_account",
+			"on_account_object",
+			"on_schema",
+			"on_schema_object",
+		},
+	},
+	"on_account_object": {
+		Type:        schema.TypeList,
+		Optional:    true,
+		ForceNew:    true,
+		Description: "Specifies the account object on which privileges will be granted ",
+		MaxItems:    1,
+		ExactlyOneOf: []string{
+			"on_account",
+			"on_account_object",
+			"on_schema",
+			"on_schema_object",
+		},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"object_type": {
+					Type:        schema.TypeString,
+					Required:    true,
+					ForceNew:    true,
+					Description: "The object type of the account object on which privileges will be granted. Valid values are: USER | RESOURCE MONITOR | WAREHOUSE | DATABASE | INTEGRATION | CONNECTION | FAILOVER GROUP | REPLICATION GROUP | EXTERNAL VOLUME",
```

Comment:
CONNECTION -> COMPUTE POOL

Reply:
And maybe in some other places, please check it

Reply:
Done